activestate.yaml

```diff
@@ -67,6 +80,27 @@ scripts:
       export TARGET_SUFFIX=${1:-}
       mkdir -p build/update
       go run scripts/update-generator/main.go -o build/update $constants.BUILD_TARGET${TARGET_SUFFIX}
+  - name: generate-update-32bit
+    language: bash
+    description: Generate update files
+    value: |
+      set -e
+      $constants.SET_ENV
+
+      export GOARCH=386
+
+      mkdir -p build/update
+      go run scripts/update-generator/main.go -o build/update $constants.BUILD_TARGET_32bit
+  - name: sign-state-tool
+    if: eq .OS.Name "Windows"
+    language: bash
+    description: Signs the state tool with the signtool executable
+    value: |
+      set -e
+
+      # This assumes that a certificate file Cert.p12 file is unpacked in the current directory
+      signtool.exe sign -d "ActiveState State Tool" -f "Cert.p12" -p ${CODE_SIGNING_PASSWD} ${constants.BUILD_TARGET}
+      signtool.exe sign -d "ActiveState State Tool (32bit)" -f "Cert.p12" -p ${CODE_SIGNING_PASSWD} ${constants.BUILD_TARGET_32bit}
```

Comment:
This is using inputs defined only in the CI config, so a bit awkward to have here. Let's move it back to the CI config as I don't want to scope creep this story too much.